x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/transport/ServerTransportFilterTests.java

```diff
@@ -85,6 +96,52 @@ public void testInbound() throws Exception {
         verify(authzService).authorize(eq(authentication), eq("_action"), eq(request), anyActionListener());
     }
 
+    public void testRemoteAccessInbound() {
+        TransportRequest request = mock(TransportRequest.class);
+        Authentication authentication = AuthenticationTestHelper.builder().internal(SystemUser.INSTANCE).build();
```

Comment:
Nit: I think this is copied from existing tests. But I don't think there is any reason that this has to be `_system`?

Reply:
Yup no good reason for this.